src/renderers/PartialViewRenderers/region/componentsInfo.js

```diff
@@ -1,361 +1,373 @@
+  }
+  return activationLabel;
 }
 
 function getComponentTypeColorClass(componentTypeId) {
-    let className = '';
-    switch (componentTypeId) {
-        case 1:
-            className = 'valuePopulation';
-            break;
-        case 2:
-            className = 'valueBuilding';
-            break;
-        case 3:
-            className = 'valueResource';
-            break;
-        case 4:
-            className = 'valueFood';
-            break;
-        case 5:
-            className = 'valueMoney';
-            break;
-        case 6:
-            className = 'valueSpecial';
-            break;
-    }
-    return className;
+  let className = "";
+  switch (componentTypeId) {
+    case 1:
+      className = "valuePopulation";
+      break;
+    case 2:
+      className = "valueBuilding";
+      break;
+    case 3:
+      className = "valueResource";
+      break;
+    case 4:
+      className = "valueFood";
+      break;
+    case 5:
+      className = "valueMoney";
+      break;
+    case 6:
+      className = "valueSpecial";
+      break;
+  }
+  return className;
 }
 
-function replaceElementAttributeContent(element, attribute, stringToReplace, newString) {
-    let attributeContent = $(element).attr(attribute);
-    $(element).attr(attribute, attributeContent.replace(stringToReplace, newString));
+function replaceElementAttributeContent(
+  element,
+  attribute,
+  stringToReplace,
+  newString
+) {
+  let attributeContent = $(element).attr(attribute);
+  $(element).attr(
+    attribute,
+    attributeContent.replace(stringToReplace, newString)
+  );
 }
 // END UTILITY FUNCTIONS
 
 $(function () {
-    //Get all component related info
-    getComponentsInfo();
-    //Get all facilities for facility ddl
-    getFacilitiesList();
-    //event handler for component display change
-    rbsComponentsDisplay_onChange();
-    //handles events for addUpdateComponent
-    addUpdateComponent_handler();
-    //handle delete component events
-    deleteComponent_handler();
-    //handle events from main page
-    pageMain_eventHandler();
-    //handle events from child component modal
-    mdlChildComponents_eventHandler();
-    //handle events from btnOpenBulkInsertComponents
-    btnOpenBulkInsertComponents_eventHandler();
-    //handle events from bulk insert page
-    bulkInsertPage_eventHandler();
+  //Get all component related info
+  getComponentsInfo();
+  //Get all facilities for facility ddl
+  getFacilitiesList();
+  //event handler for component display change
+  rbsComponentsDisplay_onChange();
+  //handles events for addUpdateComponent
+  addUpdateComponent_handler();
+  //handle delete component events
+  deleteComponent_handler();
+  //handle events from main page
+  pageMain_eventHandler();
+  //handle events from child component modal
+  mdlChildComponents_eventHandler();
+  //handle events from btnOpenBulkInsertComponents
+  btnOpenBulkInsertComponents_eventHandler();
+  //handle events from bulk insert page
```

Comment:
Could you remove all the comments

Reply:
Sure aaaa

Reply:
You haven't removed these comments

Reply:
Oh I didn't realize that was comments outside of my code scope, do you want me to just remove all comments in general?

Reply:
Nah, let's keep it local just here first